### DISCUSSION TITLE
[Question][Hw4] Typo in capability.ml

### DISCUSSION THREAD

**Discussion by kaist-kihwan** [prosyslab-classroom/cs348-information-security#702] [Q&A]
Name: Kim Kihwan

Inside of capability.ml, there's lookup function which is
```
let lookup sub obj acl =
  try
    let submap = ObjectMap.find obj acl in
    SubjectMap.find sub submap
  with _ -> []
```
This code is identical to lookup in acl.ml. I think this is kind of copy-and-paste issue.
This code raise type error, so please check.

**Comment by sujin0529** [comment]
Hi, sorry for confusion.
Capability's lookup code should be changed as below.

```
let lookup sub obj acl =
  try
    let objmap = SubjectMap.find sub acl in
    ObjectMap.find obj objmap
  with _ -> [] 
```

Thanks
